src/layouts/sidebar-sidecar/components/table-of-contents/index.tsx

```diff
@@ -17,6 +17,11 @@ const TableOfContents = ({ headings }: TableOfContentsProps): ReactElement => {
    */
   const activeSection = useActiveSection(headings, isDesktop)
 
+  // Don't render if only one item
+  if (headings.length === 1) {
+    return null
+  }
```

Comment:
🚧  I think `useActiveSection`'s second argument also needs to be updated as well so that the hook is disabled when `headings.length === 1` or `isDesktop` is `true`.